test/journeys/lib/test-helpers/recents-widget/index.js

```diff
@@ -4,8 +4,10 @@ import waitForPromise from '../../wait-for-promise';
 
 export const elements = {
   loadingScreen: '.ciscospark-loading',
-  recentsWidget: '.ciscospark-spaces-list-wrapper',
```

Comment:
lolwut.

Reply:
that class is only present when the spaces list has at least one space so i swapped to the `ciscospark-recnts-widget` instead